### DISCUSSION TITLE
Huion H430P not detected

### DISCUSSION THREAD

**Discussion by josempaii** [OpenTabletDriver/OpenTabletDriver#2004] [Q&A] [answered]
OS: Manjaro Linux x86_64 
Kernel: 5.15.16-1-MANJARO
Hello, it does not detect my h430p , I dont know whats happening because no kind of error pops up, it just says no tablet detected. I tried using DIGImend and it worked. Whats the problem?

**Comment by jamesbt365** [comment, marked as answer]
http://opentabletdriver.net/Wiki/FAQ/Linux
